packages/devtools/src/devtoolsdriver.js

```diff
@@ -133,8 +133,8 @@ export default class DevToolsDriver {
         page.setDefaultTimeout(this.timeouts.get('pageLoad'))
     }
 
-    getPageHandle () {
-        if (this.currentFrame) {
+    getPageHandle ({ isInFrame = false } = {}) {
```

Comment:
I was more thinking about that:

```suggestion
    getPageHandle (isInFrame) {
```

Reply:
got it, I've started with that, but then changed to parameter only because `getPageHandle(true)` doesn't look really informative.

Let me change it back

Reply:
@christian-bromann done